scripts/nb-tester/qiskit_docs_notebook_tester/config.py

```diff
@@ -30,6 +30,50 @@
 _set_mpl_loglevel("critical")
 """
 
+BUILT_IN_PATCHES = {
+    "qiskit-fake-provider": dedent(
+        """
+        import warnings
+        from qiskit.providers.fake_provider import GenericBackendV2
+        from qiskit_ibm_runtime import QiskitRuntimeService
```

Comment:
Can you move the builtin patches to files internal to the project for consistency? This will make it easier to edit/add builtin patches

Reply:
Thanks for reviewing!

I could put them in separate files and use [importlib](https://docs.python.org/3/library/importlib.resources.html) to read them, but we'd still need the aliases (e.g. `qiskit-fake-provider`) so users can refer to them. I feel like that would make things _more_ complex?

Reply:
From my perspective, you don't need aliases as long as the different options are documented. But if you do choose to use aliases, I think having the interface for adding a new patch be exclusively via a new file is easier to follow - i.e. if I want to add a bunch of patches into the repo I can just write the patch file, the same way I would make a custom patch. This would clean up the code for default patches as well. For instance, you could look for the patch name as a file, and if it doesn't work, check all patches in `qiskit_docs_notebook_tester/patches/`.

For instance:
`--patch { name="defaultpatch" }`
Would first search `./defaultpatch`, then `qiskit_docs_notebook_tester/patches/defaultpatch.patch`

This way I can also go to the repo and very easily see what default patches we have by going to `qiskit_docs_notebook_tester/patches/`.

Ideally I would like to be able to create new patch files as a PR into nb-tester in the future, but if it makes things too complex then I'm ok merging without it.

Reply:
Fair enough! Added in 938dea027450f1c606881928c14dd977d1fabf20